docs/clusterfuzz/architecture.md

```diff
@@ -21,35 +22,41 @@ triaging crashes, minimizing reproducers, [bisecting], and verification of fixes
 ClusterFuzz is written in Python and Go. It runs on **Linux**, **macOS**, and **Windows**.
 
 ## Requirements
-It runs on the [Google Cloud Platform](https://cloud.google.com/), and depends
+ClusterFuzz runs on the [Google Cloud Platform](https://cloud.google.com/) and depends
 on a number of services:
-- Compute Engine (Not strictly necessary. Bots can run anywhere).
+- Compute Engine (Not strictly necessary. Bots can run anywhere.)
```

Comment:
Kevin, do you understand what "bots" mean here? Should we say "Fuzzing bots" to be a bit more explicit?

Reply:
Good point! I also linked to the section below.